### PR TITLE
Created vet clinic database

### PR DESCRIPTION
**Pull request by stephnna**
Hi code Reviewers,

I created table named animals, Inserted some data into it.
Copied the above data data.sql file in the generated repository.
I then wrote queries for the several quests and took a screenshot of the results of the queries.
Finally, I copied the SQL query into the queries.sql file in the generated repository
img_1
img_2_
img_3

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,8 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animal (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', '02-03-2020', 0, TRUE, 10.23);
+INSERT INTO animal (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Agumon', '11-15-2018', 2, TRUE, 8);
```
Comment (danmainah) [inline data.sql:4]: - [x] Kindly remove one insert for Agumon you have two of them.also, ensure you insert the correct values.

File: data.sql [hunk 2]
```diff
@@ -1,5 +1,8 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animal (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', '02-03-2020', 0, TRUE, 10.23);
```
Comment (danmainah) [inline data.sql:3]: - [x] Kindy ensure you name your values correctly,this will be important to produce the correct query result.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @stephnna , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: data.sql (hunk 2 above)
Comment (stephnna) [reply data.sql:3]: Done!
Thanks!

**Review by omarsalem7 — APPROVED** [review]
Hi @stephnna 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**